client/src/forms/BlankOrData.res

```diff
@@ -1,55 +1,58 @@
-type rec t =
-  | PEventName(BlankOr.t<string>)
-  | PEventModifier(BlankOr.t<string>)
-  | PEventSpace(BlankOr.t<string>)
-  | PDBName(BlankOr.t<string>)
-  | PDBColName(BlankOr.t<string>)
-  | PDBColType(BlankOr.t<string>)
-  | PFnName(BlankOr.t<string>)
-  | PFnReturnTipe(BlankOr.t<DType.t>)
-  | PParamName(BlankOr.t<string>)
-  | PParamTipe(BlankOr.t<DType.t>)
-  | PTypeName(BlankOr.t<string>)
-  | PTypeFieldName(BlankOr.t<string>)
-  | PTypeFieldTipe(BlankOr.t<DType.t>)
+// WHAT this file seems wholly unused. Delete? or maybe it's new, and going
+// to be used soon.
```

Comment:
I think the type is still in types.res, but I had intended for this to be the new home